Remotion/Validation/UnitTests/Rules/PropertyValidationRuleTest.cs

```diff
@@ -1,30 +1,194 @@
-﻿using NUnit.Framework;
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using Moq;
+using NUnit.Framework;
+using Remotion.Reflection;
+using Remotion.Validation.Results;
 using Remotion.Validation.Rules;
 using Remotion.Validation.UnitTests.TestDomain;
+using Remotion.Validation.Validators;
 
 namespace Remotion.Validation.UnitTests.Rules
 {
   [TestFixture]
   public class PropertyValidationRuleTest
   {
     [Test]
-    [Ignore("TODO RM-5906")]
-    public void Validate ()
+    public void Validate_WithNullAsInstanceToValidate_ReturnsEmptyResult ()
     {
+      var context = new ValidationContext(null);
+
+      var validatorMock = new Mock<IPropertyValidator>();
+      validatorMock
+          .Setup(_ => _.Validate(It.IsAny<PropertyValidatorContext>()))
+          .Returns(Array.Empty<PropertyValidationFailure>());
+
+      var propertyInformationMock = new Mock<IPropertyInformation>();
+
+      Func<object, object> propertyFunc = i => i;
+      Func<object, bool> conditionFunc = i => i != null;
+
+      var rule = new PropertyValidationRule<object, int>(propertyInformationMock.Object, propertyFunc, conditionFunc, new[] { validatorMock.Object });
+
+      var res = rule.Validate(context);
+
+      Assert.That(res, Is.Empty);
     }
 
     [Test]
-    [Ignore("TODO RM-5906")]
-    public void IsActive ()
+    public void Validate_WithConditionIsFalse_ReturnsEmptyResult ()
     {
-      // consider better name than "IsActive"
+      var context = new ValidationContext("test");
```

Comment:
an object instance instead of a string, please. Reasoning is that object is intended to be a domain type. We can use e.g. `Person` from the test domain to make it more expressive.